apps/app/components/project/send-project-invitation-modal.tsx

```diff
@@ -8,9 +8,15 @@ import { useForm, Controller, useFieldArray } from "react-hook-form";
 
 import { Dialog, Transition } from "@headlessui/react";
 // ui
-import { CustomSelect, PrimaryButton, SecondaryButton, TextArea } from "components/ui";
+import {
+  Avatar,
+  CustomSearchSelect,
+  CustomSelect,
+  PrimaryButton,
+  SecondaryButton,
+} from "components/ui";
 //icons
-import { PlusIcon, XMarkIcon } from "@heroicons/react/24/outline";
+import { ChevronDownIcon, PlusIcon, XMarkIcon } from "@heroicons/react/24/outline";
```

Comment:
What's the need to import the chevron icon, there's already a prop `noChevron` to control it.